instaclustr/resource_cluster.go

```diff
@@ -56,10 +57,34 @@ func resourceCluster() *schema.Resource {
 
 			"data_centre": {
 				Type:     schema.TypeString,
-				Required: true,
+				Optional: true,
 				ForceNew: true,
 			},
 
+			"data_centres": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				ForceNew: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
```

Comment:
If you look at https://developer.instaclustr.com/#operation/extendedProvisionRequestHandler, there are many more things that can be specified when creating a data centre in a multi-dc request (e.g. provider, node size).  Should this be updated to include all of those?

Reply:
cool, thanks Chris